skylog/logger.go

```diff
@@ -0,0 +1,37 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package skylog
```

Comment:
Do we need this prefix? I can see other packages don't have `sky` or `skywalking` prefix.

Reply:
Update the logger adapter to the `skywalking` package

Reply:
Opss. I need to move back to the `skylog`, because it will cause the import cycle.

Reply:
Can't we keep log only? As the project is already skywalking, we don't use sky prefix usually.

Reply:
Ok, update the package name to `log`